documentation/Using-the-SDK/Creating-an-extended-table.md

```diff
@@ -42,34 +43,48 @@ This class __MUST__ have the following items:
                     // Paths to the Cookers that are needed for
                     // this table. This is not needed if you are
                     // using the RequiresCooker attributes.
-                    //
                 });
             );
 
-        // This method is required so that the runtime can build your
-        // table.
+        //
+        // This method, with this exact signature, is required so that the runtime can 
+        // build your table once all cookers have processed their data.
         //
         public static void BuildTable(
             ITableBuilder tableBuilder,
-            IDataExtensionRetrieval tableData
+            IDataExtensionRetrieval requiredData
         )
         {
-            // Query cooked data from dataRetrieval, and use the
+            //
+            // Query cooked data from requiredData, and use the
             // tableBuilder to build the table.
             //
         }
     }
 ````
 
-There is no restriction on the Cookers on which your Table may depend. You
-may author your own Cookers leveraging Cookers from many different plugins,
-and have your Table depend on that. As long as you have Cooked Data, you
-can create an Extended Table.
+There is no restriction on the cookers on which your `Table` may depend. You
```

Comment:
Using may a lot in these few lines makes the meaning unclear on the first read through. Instead, use words like for example to explain one example, and then offer different possibilities.